src/leveldb/README.md

```diff
@@ -49,7 +49,7 @@ mkdir build
 cd build
 cmake -G "Visual Studio 15" ..
 ```
-The default default will build for x86. For 64-bit run:
```

Comment:
This was already proposed upstream at https://github.com/google/leveldb/pull/769 and https://github.com/google/leveldb/pull/739/files - but since that repo is basically dead, you can propose it to our fork in https://github.com/bitcoin-core/leveldb-subtree instead (where it will also likely be ignored since there's an opportunity cost for spending time with these)